src/server/i18n/index.js

```diff
@@ -17,12 +17,29 @@
  * under the License.
  */
 
+import { resolve } from 'path';
+import glob from 'glob';
+import { promisify } from 'util';
 import { i18n, i18nLoader } from '@kbn/i18n';
 
+const globAsync = promisify(glob);
+
 export async function i18nMixin(kbnServer, server, config) {
-  const { translationPaths = [] } = kbnServer.uiExports;
   const locale = config.get('i18n.locale');
 
+  const groupedEntries = await Promise.all(
+    config.get('i18n.translationsScanDirs').map(async path => {
+      const entries = await globAsync('**/translations/*.json', {
```

Comment:
Can we use a smaller-scope glob here? If we use `globby` instead of glob we can use multiple paths and avoid ignoring or accidentally recursing into massive directories like node_modules:

```js
const entries = await globby(['*/translations/*.json', '*/plugins/*/translations/*.json'], {
  cwd: path
});
```